python/mxnet/optimizer.py

```diff
@@ -251,9 +251,10 @@ class SGD(Optimizer):
     param_idx2name : dict of string/int to float, optional
         special treat weight decay in parameter ends with bias, gamma, and beta
     """
-    def __init__(self, momentum=0.0, **kwargs):
+    def __init__(self, momentum=0.0, nesterov=False, **kwargs):
```

Comment:
I think it is compatible with the original sgd, can you double check that when nesterov is false it is same with the original code?